### DISCUSSION TITLE
Getting error for ext-fileinfo

### DISCUSSION THREAD

**Discussion by Jayeshzerones** [opcodesio/log-viewer#275] [Q&A]
You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with `--ignore-platform-req=ext-fileinfo` to temporarily ignore these required extensions

**Comment by arukompas** [comment]
hey @Jayeshzerones , sorry but I'm confused a little bit. Is this a reply to some other discussion, or is there a question here?
